platform/admin/ui/src/main/webapp/js/views/installer/Finish.view.js

```diff
@@ -31,17 +31,23 @@ define([
             this.navigationModel = options.navigationModel;
             this.listenTo(this.navigationModel,'next', this.next);
             this.listenTo(this.navigationModel,'previous', this.previous);
+            this.model = this.navigationModel;
         },
         onClose: function() {
             this.stopListening(this.navigationModel);
         },
         next: function() {
+            this.showLoading();
             //this is your hook to perform any validation you need to do before going to the next step
             this.navigationModel.nextStep();
         },
         previous: function() {
             //this is your hook to perform any teardown that must be done before going to the previous step
             this.navigationModel.previousStep();
+        },
+        showLoading: function(){
```

Comment:
What is this function doing? What causes the overlay to toggle off and the main content to come back. Also, these things look like they should trigger off of some sort of model change event and simply re-render the view.